node-runner-cli/tests/fixtures/docker-compose.yaml

```diff
@@ -6,7 +6,7 @@ services:
     environment:
       JAVA_OPTS: --enable-preview -server -Xms8g -Xmx8g  -XX:MaxDirectMemorySize=2048m
         -XX:+HeapDumpOnOutOfMemoryError -XX:+UseCompressedOops -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts
-        -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector
+        -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.log_util.log4j.core.async.AsyncLoggerContextSelector
```

Comment:
Same here as mentioned in below comment . This is java class

Reply:
yep. reverting